rustfmt.toml

```diff
@@ -0,0 +1,24 @@
+# all formatting is executed and checked on the developer's side
```

Comment:
I would be very happy if you use the following rustfmt.toml file configuration and re-format the entire PR: https://github.com/Robbepop/pdsl/blob/master/.rustfmt.toml

This format config is well suited for reviewing PRs.

Reply:
Also I am thinking about adding the following extra compiler warnings and lints to improve the readability of the entire code base so that it becomes easier for users not familiar with the code base to hack on. After your changes and the new release we might see more users.

https://github.com/Robbepop/pdsl/blob/master/model/src/lib.rs#L19

Reply:
Besides that I am planning to extend the CI pipeline to include rustfmt checks as well as clippy lints.

Reply:
So it would be awesome if you could already check for clippy warnings in your PR if you didn't do that already. ;)